src/PSR7/Validators/PathValidator.php

```diff
@@ -49,8 +49,9 @@ private function validateRequest(OperationAddress $addr, RequestInterface $messa
         $validator        = new SchemaValidator($this->detectValidationStrategy($message));
 
         foreach ($pathParsedParams as $name => $value) {
+            $parameter = RequestParameter::fromSpec($specs[$name]);
```

Comment:
I think with new usages this class should have new name. `SerializedParameter` maybe?

Reply:
Renamed it